src/tools/io.lua

```diff
@@ -17,7 +17,7 @@ function _M.read_file(path)
 end
 
 function _M.write_to_file(path, value)
-  local file = io.open(path, "w")
+  local file, err = io.open(path, "w")
```

Comment:
This is a problem that I've encountered, for some reason the permission on the file where wrong and it was throwing:

```
lua: /usr/local/share/lua/5.1/kong/tools/io.lua:22: attempt to index local 'file' (a nil value)
stack traceback:
	/usr/local/share/lua/5.1/kong/tools/io.lua:22: in function 'write_to_file'
	/usr/local/share/lua/5.1/kong/cli/config.lua:51: in main chunk
	[C]: in function 'require'
	bin/kong:38: in main chunk
	[C]: ?
```

Problem is: `write_to_file` is also being used by other functions (is it?) so throwing `error()` might crash the server.